workers/ohsome_quality_analyst/indicators/mapping_saturation/metrics.py

```diff
@@ -0,0 +1,29 @@
+"""Collection of common error metrics as functions."""
+
+import numpy as np
+from numpy import float64, ndarray
+
+
+def me(actual: ndarray, predicted: ndarray) -> float64:
```

Comment:
Of all those functions only `mae` ("Mean Absolute Error") is used at the moment.

I thought it would be nice to have those errors implemented for the future. In fact at first we have not been sure about the metric for model selection.

On the other hand `sklearn` and with it access to various metrics will be available for OQT soon. 

I will remove the other metrics.